hs_core/views/resource_rest_api.py

```diff
@@ -261,6 +261,11 @@ def get_queryset(self):
             raise ValidationError(detail=resource_list_request_validator.errors)
 
         filter_parms = resource_list_request_validator.validated_data
+
+        if filter_parms['coverage_type']:
```

Comment:
I guess we need to change this to: `if 'coverage_type'` in `filter_params:`

Reply:
good call thanks